code/datums/autolathe/general.dm

```diff
@@ -122,3 +122,7 @@
 	name = "handcuffs"
 	path =/obj/item/weapon/handcuffs
 	hidden = 1
+
+/datum/category_item/autolathe/general/holocollar
```

Comment:
Put this in general_vr.dm

Reply:
Like just add an entirely new file with that? Hmm....

Reply:
![yes](https://user-images.githubusercontent.com/6484985/50726335-c163c680-10d9-11e9-832f-526a6d1aa0a4.jpg)

(Prevents conflicts is why.)